### ISSUE TITLE
Add helper class for flex-center

### ISSUE THREAD

**Issue by kwalker3690** [optimizely/oui#342]
We have this great mixin:
```@mixin flex-center {
  display: flex;
  justify-content: center;
  align-items: center;
}```

but no corresponding helper class, so to recreate you need to use three classes currently. Would be great to have just one class that centers an object like this mixin.

**Comment by daverau-optimizely** [comment]
Good call @kwalker3690 — I've had opportunities to use this as well.

**Comment by danoc** [comment]
This has been merged and released in v11.0.0.
